code/lib/theming/src/global.ts

```diff
@@ -11,6 +12,7 @@ interface Return {
 export const createReset = memoize(1)(
   ({ typography }: { typography: Typography }): Return => ({
     body: {
+      color: color.defaultText,
```

Comment:
Is it getting set elsewhere right now?

Reply:
@domyen It seems to not be set anywhere, though [line 104 of that file](https://github.com/storybookjs/storybook/blob/9aab2c6bc78cc5f4c686ef59ffa35f51b6d47e69/code/lib/theming/src/global.ts#L104) looks like it MIGHT be a place that is supposed to set it, but isn't.